backends/graphics3d/ios/ios-graphics3d.cpp

```diff
@@ -90,6 +96,11 @@ void iOSGraphics3dManager::initSurface() {
 void iOSGraphics3dManager::deinitSurface() {
 	glDeleteFramebuffers(1, &_glFBO);
 	glDeleteRenderbuffers(2, _glRBOs);
+	if (_offScreenRendering) {
```

Comment:
So, here, the _glRBO is destroyed by us.
But at init, the _glRBO is created by the platform code.
And below,  if I understand correctly, the _glRBO is destroyed in the platform part of the code and replaced by another one.
I think it would be better to handle all of this in platform code (so, in destroyOpenGLContext).
That let you avoid storing the _glRBO like you did.